### PR TITLE
Update strings to match the new glossary.

### PR DESCRIPTION
**Pull request by mwichary**
@abe33

***PLEASE DO NOT MERGE***

Question: Is renaming commands okay? Will it have consequences if people already mapped them to other shortcuts? If so, is there a way to safely rename them?

New functionality: Re-saving some configs under new names and removing old ones. 

Otherwise just string changes.

This is part of a bigger effort documented here: https://kite.quip.com/eGu8AecvGYze/New-UI-strings-deployment

### CODE REVIEW

**Comment by mwichary** [comment]
(Also cc @jansorg as an fyi.)

File: lib/kite.js [hunk 1]
```diff
@@ -33,9 +33,18 @@ module.exports = {
       }
     }
 
-    // little thing to take out legacy config name value so it won't show up on settings
+    // Deal with legacy setting names
+    // March 2018
     atom.config.unset('kite.actionWhenKiteFixesCode');
 
+    // April 2018
+    atom.config.set('kite.searchIconPosition', atom.config.get('kite.activeSearchPosition'));
```
Comment (abe33) [inline lib/kite.js]: We probably don't want to do it that way. Currently, you're overriding whatever value the user has set, everytime, with either the old value or `undefined` after the first run of these lines. That's definitely not what we want here. 

The proper way to do that is as follow: 

```js
if (atom.config.get('kite.activeSearchPosition') != undefined) {
  atom.config.set('kite.searchIconPosition', atom.config.get('kite.activeSearchPosition'));
  atom.config.unset('kite.activeSearchPosition');
}
```

File: lib/kite.js [hunk 2]
```diff
@@ -33,9 +33,18 @@ module.exports = {
...
+    atom.config.unset('kite.activeSearchPosition');
+
+    // April 2018
+    atom.config.set('kite.showWelcomeNotificationOnStartup', atom.config.get('kite.showDocsNotificationOnStartup'));
```
Comment (abe33) [inline lib/kite.js]: Same as above

File: lib/elements/kite-active-search.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ const GETTING_STARTED = [
 
 class KiteActiveSearch extends HTMLElement {
   static initClass() {
-    atom.commands.add('kite-active-search', {
+    atom.commands.add('kite-search', {
```
Comment (abe33) [inline lib/elements/kite-active-search.js]: You're breaking things here, that string is the element name where these commands apply. Without changing the element name in its definition (as well as the associated styles and file name for consistency) this won't work.

Comment (mwichary) [reply lib/elements/kite-active-search.js]: Oh, yikes. Good to know! I will redo.

File: lib/kite.js (hunk 1 above)
Comment (mwichary) [reply lib/kite.js]: Ugh. Yes. Sorry.

**Comment by mwichary** [comment]
Okay, I made the requested changes and tested search to make sure it still works.

/cc @abe33 if you want to look at it (last commit) again?

File: lib/elements/kite-active-search.js (hunk 3 above)
Comment (mwichary) [reply lib/elements/kite-active-search.js]: Renamed everything from active-search to search.

**Pull request merged**